packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -356,6 +357,21 @@ const Dispatcher: DispatcherType = {
   useId,
 };
 
+// create a proxy to throw a custom error
+// in case future versions of React adds more hooks
+const DispatcherProxyHandler = {
+  get(target, prop, _receiver) {
+    if (target.hasOwnProperty(prop)) {
+      return Reflect.get(...arguments);
```

Comment:
Curious why you need `Reflect.get(...arguments)` here. Would 
```
const DispatcherProxy = new Proxy(Dispatcher, {
   get(target, prop) {
     if (target[prop]) {
          return target[prop];
     } 
     throw error;
  }
})
```

not work?

Reply:
It's actually the same. I guess I'll change to `target[prop]` for better readability